core/src/main/java/tc/oc/pgm/modules/MultiTradeMatchModule.java

```diff
@@ -24,9 +24,10 @@ public MultiTradeMatchModule(Match match) {
     this.logger = PGM.get().getGameLogger();
   }
 
-  @EventHandler(priority = EventPriority.NORMAL, ignoreCancelled = true)
+  @EventHandler(priority = EventPriority.NORMAL)
   public void onInteract(PlayerInteractEntityEvent event) {
     if (event.getRightClicked() instanceof Villager) {
+      if (event.isCancelled()) return;
```

Comment:
What's the reasoning for this change?

Reply:
Reason behind this is to ensure that regular villager trades don't open when the player clicks an interactive shop keeper. 

I believe I mistakenly left the `event.isCancelled` in here, since the removal of `ignoreCancelled` should be all that's needed. Good catch!